src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2187,11 +2187,12 @@ public T block(Duration timeout) {
 	 *
 	 * @return an alternating {@link Mono} on source onError
 	 *
-	 * @see Flux#onErrorResumeWith
+	 * @deprecated Use {@link #otherwise(Function)} instead.
```

Comment:
should be `#switchOnError` not `#otherwise`. Maybe also indicate it will be removed in 3.1.0.RELEASE (or any pre-release after M1)?